### PR TITLE
Make srivo kind up lane mandatory

### PR DESCRIPTION
**Pull request by qinqon**
We must not merge PRs that breaks sriov provider.

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ presubmits:
   - name: check-up-kind-1.17-sriov
     annotations:
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
-    always_run: false
+    always_run: true
     optional: true
```
Comment (oshoval) [inline github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:7]: please make it `optional: false`

**Review by oshoval — CHANGES_REQUESTED** [review]

**Review by ormergi — COMMENTED** [review]
I know this is not the PR scope but this label is a left over and I think that if someone will use rehearsal plugin it will run this job as well https://github.com/kubevirt/project-infra/pull/733/files#diff-eeeb062464da267e259438c2a43d08bf1dd3d44cf8c1e629addbeaa2a34454abR17
Could you please remove it?

**Comment by qinqon** [comment]
@ormergi that's out of scope of the PR.

**Review by phoracek — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**